src/main/resources/application.properties

```diff
@@ -121,7 +121,7 @@ spring.application.name=ccd-data-store
 http.client.max.total=${HTTP_CLIENT_MAX_TOTAL:100}
 http.client.seconds.idle.connection=${HTTP_CLIENT_SECONDS_IDLE_CONNECTION:120}
 http.client.max.client_per_route=${HTTP_CLIENT_MAX_CLIENT_PER_ROUTE:20}
```

Comment:
you might need to update demo.tfvars (ASE) ?

Reply:
👍